src/main/java/seedu/address/logic/parser/AddModuleCommandParser.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_CODE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE_TITLE;
+import static seedu.address.logic.parser.ParserUtil.arePrefixesPresent;
+import static seedu.address.model.module.Module.EMPTY_MODULE_TITLE;
+
+import seedu.address.logic.commands.AddModuleCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.module.Module;
+import seedu.address.model.module.ModuleCode;
+import seedu.address.model.module.ModuleTitle;
+
+/**
+ * Parses input arguments and creates a new AddCommand object
```

Comment:
```suggestion
 * Parses input arguments and creates a new {@code AddModuleCommand} object
```
I believe there is a typo here, as well as in the javadocs below.

Reply:
Yes, there is. Thanks for spotting it.